src/AzureAuth/CommandMain.cs

```diff
@@ -52,6 +52,11 @@ public class CommandMain
         private Alias tokenFetcherOptions;
         private ITokenFetcher tokenFetcher;
 
+        /// <summary>
+        /// MutexTimeout is the timeout of Mutex to prevent infinite waiting.
+        /// </summary>
+        private TimeSpan mutexTimeout = TimeSpan.FromMinutes(15);

```

Comment:
```suggestion
        private TimeSpan promptMutexTimeout = TimeSpan.FromMinutes(15);

```
Might be worth a little more specificity here. This particular mutex is guarding the number of user prompts.